app/javascript/chat/components/createChatModal.jsx

```diff
@@ -0,0 +1,65 @@
+import { h } from 'preact';
+import { useState } from 'preact/hooks';
+import PropTypes from 'prop-types';
+import { Modal } from '../../crayons/Modal';
+import { createChannel } from '../actions/chat_channel_setting_actions';
+const CreateChatModal = ({
```

Comment:
documentation (blocking): Document the component as it's exported.

nitpick: Arrow functions are only necessary when you need to deal with scope, i.e. `this`. In the case of function components, there is no this. Everything is functions.